### ISSUE TITLE
Cannot find Asteroid, keep warping

### ISSUE THREAD

**Issue by nexty5870** [Viir/bots#31]
Hi everyone, 

have try the bot yesterday, it seems being looping in warping saying that he cannot find asteroid on the belt, 

I have my overview set to Mining so asteroid belt are being display with the closest range as per the note

I have try to manually start the mining process in one of those belt, resume the bot and it was running properly (meaning the bot was able to detect the mining progress and drop off to station )

Does a specific fitting is required maybe? Im having Miner I laser at the present

**Comment by Viir** [comment]
Maybe the asteroids look slightly different in your game client. Such an issue is easy to fix if you share a sample from your game client.
When you link the session recording archive of your session, I can take a look. You can upload the file directly here in the GitHub issue. To get the archive for your session, see the guide at https://to.botengine.org/guide/how-to-report-an-issue-with-an-app-or-request-a-new-feature#getting-the-session-recording-archive

**Comment by nexty5870** [comment]
Thank you for getting back to me that quick! 

attached one of the session log
[session-2020-12-28T00-47-29-4e91d7.zip](https://github.com/Viir/bots/files/5747636/session-2020-12-28T00-47-29-4e91d7.zip)

event 146 you can see the Asteorid available to mine there

Thank again!

**Comment by Viir** [comment]
In the archive `session-2020-12-28T00-47-29-4e91d7.zip`, your game client's reading looks like the 'Name' column in the overview is not visible.

Does it work when you make this column visible in the overview?

**Comment by nexty5870** [comment]
that make the trick actually! thank a lot

have now order some credit as well - amazing dev, thank you!

**Comment by Viir** [comment]
I expanded the bot to avoid it skipping asteroid belts in this scenario: https://github.com/Viir/bots/commit/dea59f373d7be5f3908ec1e2ff24a5309fbb7617
For the scenario `session-2020-12-28T00-47-29-4e91d7.zip`, this version shows a new message instead, for example, in event 110:

```txt
Session performance: times unloaded: 0, volume unloaded / m³: 0
---
Current reading: Ore hold filled 0%. Shield HP at 100%. Found 2 mining modules. I see the drones window: In bay: 2, in local space: 0.
+ I see ship UI and overview, undocking complete.
++ The 'Name' column in the overview window seems disabled.
+++ Please configure the overview to show objects names.
++++ I am stuck here and need help to continue.
+++++ Wait
```

**Comment by mikeysayer** [comment]
> In the archive `session-2020-12-28T00-47-29-4e91d7.zip`, your game client's reading looks like the 'Name' column in the overview is not visible.
> 
> Does it work when you make this column visible in the overview?



I have the same issue and I have tried to apply this fix, and realised the “name” column is already in the overview. 

if theres a way for me to send you the log (and how to) let me know! Would love for this to be fixed

**Comment by Viir** [comment]
> if theres a way for me to send you the log (and how to) let me know! Would love for this to be fixed

The easiest way would be to add the log (session recording archive) to this issue thread. I can drag & drop a file into the text box here, and that gets uploaded then so other people can access it.
I tested this using the Chrome browser on Windows 10.

Does this way to upload work for you?

**Comment by mikeysayer** [comment]
> > if theres a way for me to send you the log (and how to) let me know! Would love for this to be fixed
> 
> The easiest way would be to add the log (session recording archive) to this issue thread. I can drag & drop a file into the text box here, and that gets uploaded then so other people can access it.
> I tested this using the Chrome browser on Windows 10.
> 
> Does this way to upload work for you?

[session-2021-01-18T10-21-07-538048.zip](https://github.com/Viir/bots/files/5832137/session-2021-01-18T10-21-07-538048.zip)

**Comment by Viir** [comment]
Yes, that worked. I am looking at event 66 in this recording. In this case, it seems like there was no asteroid remaining in the belt. The solution is to find a solar system with asteroids remaining in the belts (or another mining site you can bookmark). You need to start the bot in a solar system that has asteroids. If you saw asteroids outside the overview window, the solution would be to make them visible in the overview.
